index.html

```diff
@@ -65,19 +65,85 @@ <h1>Hello, world!</h1>
       <!-- Example row of columns -->
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <h1>Poems</h1>
```

Comment:
I agree with the changes you made from the <h2> tag to the <h1> tag. It definitely makes it stand out more.